airbyte-db/lib/configs_db_schema.md

```diff
@@ -0,0 +1,359 @@
+# Configs Database Schema Draft
+
+This is a temporarily file that documents the design of the new configs database. Currently everything lives in the `airbyte_configs` table. We want to normalize it to one table per config type. Relevant issue: [#5479](https://github.com/airbytehq/airbyte/issues/5479).
+
+## Convention
+- The tables are as similar to the YAML definition as possible.
+  - Table names are in snake case and singular form.
+  - Column names are in snake case.
+- Use ANSI SQL as much as possible so that we are not locked in Postgres.
```

Comment:
Alright, make sense, let's commit to Postgres then and leverage all the features there.

jOOQ cannot help us because if the underlying database does not support a feature, jOOQ cannot go around it.

Reply:
I will change the `id` column to UUID, and replace string enums with enum types.

Reply:
awesome sounds good!